### PR TITLE
Development

### PR DESCRIPTION
**Pull request by nikasulo**
# Description
 This pull request contains tests written for the Player Class, Game Class and Board Class of the Tic Tac Toe

# How was it done?
The tests where written using Rspec

# How does it work?
Methods and Classes were described using the *describe* keyword and *it* was used to simulate expected behaviour for different Edge cases.

### CODE REVIEW

File: spec/board-spec.rb [hunk 1]
```diff
@@ -0,0 +1,70 @@
+require '../lib/game.rb'
```
Comment (maelfosso) [inline spec/board-spec.rb]: Please, use `require_relative` with the relative path

Comment (maelfosso) [reply spec/board-spec.rb]: That rule should also be applied in other `*.spec` files

File: spec/board-spec.rb [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+describe '::Board' do
+  let(:board) {Board.new}
+  let(:show_board) {board.show_board}
```
Comment (maelfosso) [inline spec/board-spec.rb:5]: **`OPTIONAL`** Not needed.

File: spec/board-spec.rb [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  
+  
+  describe '#show_board' do
+
+    it 'prints the first row' do
+      show_board
```
Comment (maelfosso) [inline spec/board-spec.rb]: **`OPTIONAL`** It's not suggested to print in testing. I suggest you remove `show_board`.

File: spec/board-spec.rb [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+      board.move = 9
+      expect(board.draw?).to eql(true)
+    end
+
+    it 'only signifies draws' do
+      board.move = 5
+      expect(board.draw?).to eql(false)
+    end
+  end
+end
```
Comment (maelfosso) [inline spec/board-spec.rb:67]: A beautiful way of using Rspec statement.

**Review by maelfosso — CHANGES_REQUESTED** [review]
Hello coders,

I enjoyed reading your code. The way you use RSpec statement is really nice!!!
There are optional changes I suggest you do. 
The change related to use `require_relative` is needed to make your code run. So please, do it and submit a new code review request.

Thanks

File: spec/board-spec.rb (hunk 1 above)
Comment (nikasulo) [reply spec/board-spec.rb]: Thanks, I just read about the difference in the require, require_relative and require_all methods. This makes sense

File: spec/board-spec.rb (hunk 2 above)
Comment (nikasulo) [reply spec/board-spec.rb:5]: Okay, what this test actually does is makes sure that the board has 3 rows. It doesn't actually print. What do you think?

File: spec/board-spec.rb (hunk 3 above)
Comment (nikasulo) [reply spec/board-spec.rb]: Noted with thanks.

File: spec/board-spec.rb (hunk 4 above)
Comment (nikasulo) [reply spec/board-spec.rb:67]: Thanks Mael.

**Comment by bellom** [comment]
## Currently reviewing this PR

**Review by bellom — APPROVED** [review]
Hi Team,
Great work making all the necessary corrections from the previous review.
Kindly be aware that you need to do the following in the next project to be able to get your project approved.
1. Add `linter`
2. Use of `Git Flow`
Always read the pathwright instructions before starting the project to avoid you sending another code review, according to the pathwright you are to use Git Flow not GitHub Flow.
Short Explanation Hint: 
`GitHub Flow`: Pull Requests is raise from `feature branches to the master`.
`Git Flow`: Pull Requests is raise from `feature branches to the development`.
_read more on the pathwright._
#### Congratulation.
Goodluck in your next project

**Comment by nikasulo** [comment]
Hello @bellom, thank you for the review, however please check again as we used the correct flow. We raised all pull requests from feature branches to development and development to master.
